pkg/sql/pgwire/pgerror/main_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: Andrei Matei (andreimatei1@gmail.com)
+package pgerror
+
+import (
+	"os"
+	"testing"
+
+	// Import the logging package so that tests in this package can accept the
+	// --verbosity flag and friends. As of 03/2017, no test in this package needs
+	// logging, but we want the test binary to accept the flags for uniformity
+	// with the other tests.
+	_ "github.com/cockroachdb/cockroach/pkg/util/log"
```

Comment:
are there any tests in this package? if yes, you can put this in those test files rather than here, and avoid defining TestMain.